### PR TITLE
Delete k8s import

### PR DESCRIPTION
**Pull request by jooseppi-luna**
# Description
Adjusting function name and imports (removing k8s import) to reflect changes in [gofsutil PR #8](https://github.com/dell/gofsutil/pull/8). 

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

Installed diver with image built from this code, confirmed health monitor is still functional and propery reporting creation and deletion of volumes

### CODE REVIEW

**Review by bharathsreekanth — APPROVED** [review]
Looks ok to me, but wait for approvals from code experts in this domain before merge.

File: go.mod
Comment (JacobGros) [inline go.mod:41]: Can we remove the giant replace block now that the kubernetes import is gone?

File: .github/workflows/actions.yaml
Comment (JacobGros) [inline .github/workflows/actions.yaml:19]: Does this change mean gofsutils is being checked for linting/format/vetter as well?
If so, I don't think we want to do this- It could lead to our PRs bring blocked due to a change someone else made in a different repo entirely. 
Gofsutils/other repos have their own github actions in place. 

If only the driver is being checked- then we're fine and can ignore this comment

**Review by JacobGros — CHANGES_REQUESTED** [review]

**Review by bharathsreekanth — APPROVED** [review]

**Review by bharathsreekanth — CHANGES_REQUESTED** [review]
Pending changes still, I accidentally approved earlier. pls ignore my approval.

File: Makefile
Comment (nb950) [inline Makefile:47]: does '-race' work on now ? i was told adding race flag caused errors

File: .github/workflows/actions.yaml
Comment (nb950) [reply .github/workflows/actions.yaml:19]: @JacobGros if you view file you see 4 blocks , this seems to be ok

Comment (jooseppi-luna) [reply .github/workflows/actions.yaml:19]: > Does this change mean gofsutils is being checked for linting/format/vetter as well? If so, I don't think we want to do this- It could lead to our PRs bring blocked due to a change someone else made in a different repo entirely. Gofsutils/other repos have their own github actions in place.
> 
> If only the driver is being checked- then we're fine and can ignore this comment

Yeah, but I don't think there is a way to get around it? If we don't include gofsutil here, the checks fail because it's not there. I was concerned about failing our tests if linting fails in gofsutil, but they just had PR checks added to the repo, so the odds of that happening should be extremely low. So I don't think it's a huge concern?

Comment (JacobGros) [reply .github/workflows/actions.yaml:19]: Hmm, could we exclude the gofsutil directory? If not, we can keep it as is.

**Review by JacobGros — APPROVED** [review]
LGTM, but I think it would be a good idea to look into excluding gofsutil from lint/ formatter check

**Review by bharathsreekanth — APPROVED** [review]

File: .github/workflows/actions.yaml
Comment (nb950) [reply .github/workflows/actions.yaml:19]: ./... does not match packages in the vendor directory 
so checkout code you dont want to check into a folder named vendor

**Pull request merged**
